handsontable/src/editors/numericEditor/__tests__/numericEditor.spec.js

```diff
@@ -27,6 +27,47 @@ describe('NumericEditor', () => {
     ];
   };
 
+  it('should return true in the `isOpened` after open the numeric editor', async() => {
+    handsontable({
+      columns: [
+        {
+          type: 'numeric'
+        }
+      ],
+    });
```

Comment:
No need to change that for this PR, but in the future, it would be great to keep table configurations as compact as possible to avoid excessive vertical growth.
```suggestion
    handsontable({
      type: 'numeric',
    });
```